src/Pages/Volunteer.jsx

```diff
@@ -448,6 +463,156 @@ export const Volunteer = () => {
+                  </FormHelperText>
+                )}
+                <Controller 
+                  control={control}
+                  name="acknowledges_animal_unknown_behavior"
+                  rules={{ validate: (value) => value === "Yes" || "Response is required" }}
+                  render={({ field: { value, onChange }}) => (
+                    <FormControlLabel
+                      sx={{ alignItems: "flex-start", pt:1, mb: 1 }}
+                      control={
+                        <Checkbox
+                          sx={{ mt: -1 }}
+                          checked={value === "Yes"}
+                          onChange={onCheckboxChange(onChange)}
+                        />
+                      }
+                      label="That I acknowledge that some of the animal residents that currently reside at RSW&copy; are formerly abused/neglected animals whose behavior, temperament, and health are unknown.*"
+                    />
+                  )}
+                />
+                {errors.acknowledges_animal_unknown_behavior && (
+                  <FormHelperText 
+                    error
+                    sx={{ ml: 4, mb: 1 }}
+                  >
+                    {errors.acknowledges_animal_unknown_behavior?.message}
+                  </FormHelperText>
+                )}
+                <Controller 
+                  control={control}
+                  name="acknowledges_rsw_not_responsible"
+                  rules={{ validate: (value) => value === "Yes" || "Response is required" }}
+                  render={({ field: { value, onChange }}) => (
+                    <FormControlLabel
+                      sx={{ alignItems: "flex-start", pt:1, mb: 1 }}
+                      control={
+                        <Checkbox 
+                          sx={{ mt: -1 }}  
+                          checked={value === "Yes"}
+                          onChange={onCheckboxChange(onChange)}
+                        />
+                      }
+                      label="That I acknowledge that RSW&copy; is not responsible for any and all damage, injury claims, and expenses of whatever nature arising from my participation as a Volunteer for RSW&copy;. I agree to hold RSW&copy;, its agents, successors, and assigns harmless.*"
+                    />
+                  )}
+                />
+                {errors.acknowledges_rsw_not_responsible && (
+                  <FormHelperText 
+                    error
+                    sx={{ ml: 4, mb: 1 }}
+                  >
+                    {errors.acknowledges_rsw_not_responsible?.message}
+                  </FormHelperText>
+                )}
+              </FormGroup>
+              <FormHelperText>
+                I agree to allow RSW&copy; to perform a background check on myself, which may include, but not be limited to, a check of my references, a criminal conviction check, and a credit check.
```

Comment:
They don't list references in this form, so I wonder if we should take the "a check of my references" out

Reply:
Good point! I'll remove that.